docs/examples/multi_modal/gemini.ipynb

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
<p>do we need to specify a version for <code>google-generativeai</code> pkg </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
we do (0.3.0) - but the package is published now, and the constraints are the same. can't hurt to add it though.

Reply:
done